test/file-store.js

```diff
@@ -0,0 +1,59 @@
+/* global describe, it */
```

Comment:
Maybe it's the same in all files, but this could be `/* eslint-env mocha */` instead

Reply:
Awesome! This is my bad, I didnt know that was a thing.  We should do this.